### PR TITLE
#174566481 Setup tdd environment with mocha/chai

### PR DESCRIPTION
**Pull request by Extremophile-ai**
## What does this PR do
   This PR sets up a test driven development (tdd) environment with Mocha/chai for Know Africa.

## Description of tasks to be completed?
 - Set up tdd environment with Mocha/chai to test cod
  - Install Mocha, chai, chai-http from npm
  - Configure package.json with a test script

## How should this be manually tested/checked?
- Clone the PR
- Cd into the branch `ch-set-up-tdd-environment-174566481`
- Install the dependencies using `npm install`
- Run the test on the terminal using `npm run test`
It should show 1 passing

## What are the relevant pivotal tracker stories?
[#174566481](https://www.pivotaltracker.com/story/show/174566481)

### CODE REVIEW

**Review by Fiyin-Anne — APPROVED** [review]

**Comment by codeliezel** [comment]
@Extremophile-ai re-write your PR comment to make it better

**Review by bellogo — APPROVED** [review]
tests ran successfully

**Comment by Extremophile-ai** [comment]
> @Extremophile-ai re-write your PR comment to make it better

@codeliezel  I've done that. Please take a look at it again

**Review by codeliezel — COMMENTED** [review]
set up the coverage file by installing dependencies like nyc, etc and add the flags to the test script.
also, the test you wrote should be in another file in the tests file, while you import that file into the index.
Good job.

**Review by codeliezel — CHANGES_REQUESTED** [review]
@Extremophile-ai  add the reported text file with the HTML file, I expected you to have RESEARCHED and git ignored the coverage folder, also don't name that file `test.js` give it a better name.

good job

**Review by Fiyin-Anne — APPROVED** [review]

File: src/tests/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import describe from "./index-test";
```
Comment (codeliezel) [inline src/tests/index.js:1]: this should be `import "./index-test"`

Comment (Extremophile-ai) [reply src/tests/index.js:1]: Ok

**Review by bellogo — APPROVED** [review]

**Review by Augusta-E — APPROVED** [review]
kindly put the backticks to indicate the codes

**Comment by Extremophile-ai** [comment]
> kindly put the backticks to indicate the codes

Done. Thanks!

**Review by codeliezel — COMMENTED** [review]
Good job

**Review by Fiyin-Anne — APPROVED** [review]

**Pull request merged**